bciers/apps/reporting/src/app/components/activities/uiSchemas/schemaMaps.tsx

```diff
@@ -0,0 +1,41 @@
+type UiSchemaMap = {
+  [key: string]: string;
+};
+
+type EmptyWithUnits = { units: [{ fuels: [{ emissions: [{}] }] }] };
+type EmptyWithFuels = { fuels: [{ emissions: [{}] }] };
+type EmptyOnlyEmissions = { emissions: [{}] };
+
+type DefaultEmptySourceTypeMap = {
+  [key: string]: EmptyWithUnits | EmptyWithFuels | EmptyOnlyEmissions;
+};
+
+// Activity slug & matching uiSchema
+export const uiSchemaMap: UiSchemaMap = {
+  gsc_excluding_line_tracing: "gscUiSchema",
+  gsc_solely_for_line_tracing: "gscUiSchema",
+  gsc_other_than_non_compression: "gscOtherThanNonCompression",
+  gsc_non_compression: "gscNonCompressionNonProcessingUiSchema",
+  fuel_combustion_by_mobile: "fuelCombustionMobileUiSchema",
+  hydrogen_production: "hydrogenProduction",
+  pulp_and_paper: "pulpAndPaperUiSchema",
+  refinery_fuel_gas: "refineryFuelGasUiSchema",
+  carbonate_use: "carbonatesUseUiSchema",
```

Comment:
should we import the schemas right here instead of just having the file name? this way we don't have to do a dynamic import in the `ActivityForm` component